pages/cloudflare/howtos/NextAuth.mdx

```diff
@@ -28,3 +33,18 @@ Kudos to Arnav Gupta ([`@arnavgupta00`](https://github.com/arnavgupta00)) for co
 You can find an example of this on his [example repository](https://github.com/arnavgupta00/deployment-cf-workers-prisma-nextauth).
 
 Related issues: [`workers-sdk#206`](https://github.com/opennextjs/opennextjs-cloudflare/issues/206) and [`workerd#3277`](https://github.com/cloudflare/workerd/issues/3277).
+
+### Solving issues in local dev
+
+When trying to access bindings (for instance to use [D1](/d1/) as database) depending on your implementation, you might run into:
```

Comment:
the d1 link is not pointing to anywhere, I would suggest just removing the sentence entirely?

```suggestion
When trying to access Cloudflare bindings depending on your implementation, you might run into:
```